cartpole_en.rst

```diff
@@ -0,0 +1,218 @@
+Cartpole
+~~~~~~~~~~~~~~~~~~
+
+Overview
+=======
+The inverted pendulum problem is a classic control problem in reinforcement learning. Cartpole is a discrete control task in the inverted pendulum problem. In the game there is a car with a pole on it. The cart slides side to side on a smooth and frictionless track to keep the pole upright. As shown below.
+
+.. image:: ./images/cartpole.gif
+   :align: center
+
+Install
+====
+
+installation method
+--------
+
+The Cartpole environment is built into the gym, and you can install the gym directly. Its environment id is \``CartPole-v0``\.
```

Comment:
The Cartpole environment is built into the gym, and you can install the gym directly. Its environment id is \ ``CartPole-v0`` \.